core/src/main/java/fj/P2.java

```diff
@@ -1,12 +1,13 @@
 package fj;
 
-import static fj.Function.*;
```

Comment:
Can you restore the import order as before?

Reply:
Ok, just did.